### DISCUSSION TITLE
A loud "Clack" sound when a relay is thrown

### DISCUSSION THREAD

**Discussion by mygamingaccount** [FakeFishGames/Barotrauma#11260] [Ideas and suggestions]
Why don't Barotrauma relays make a noticeable sound?

To avoid spamming audio, only relays that have a power consuming device connected should make the sound.

**Comment by girlyguppy** [comment]
that would be awesome, but lots of circuits use spammy relays and therefore would create a terrible noise machine

it should be toggleable in item settings

**Reply by mygamingaccount** [reply]
There are always better, more efficient alternatives for fast switching relays used in a logic context, people just don't know about them.

But that's a great point anyway.
Options like that are a no-go. It leads to a clunky UI.

I don't think this is something that can be fixed with clever automation. Oh well.

**Reply by mygamingaccount** [reply]
Or perhaps, if there was a *solid-state* relay that didn't clack? :laughing:

**Reply by girlyguppy** [reply]
I mean something that can be toggled in the sub editor. It'd be one more button in a UI with 60 buttons already, so I don't think that'd be much of an issue

**Reply by mygamingaccount** [reply]
I got it!
How about the loudness being determined by the load?
If you're switching thousands of kilowatts, there's a clack.
If you're using the relay for logic, there's no load, so no clack.

**Reply by girlyguppy** [reply]
That could work
